api/features/tasks.py

```diff
@@ -50,6 +57,39 @@ def trigger_feature_state_change_webhooks(
         )
     )
 
+    if hasattr(instance.environment.project.organisation, "github_config"):
```

Comment:
We need tests for this code

Reply:
This code is being tested here https://github.com/Flagsmith/flagsmith/pull/3188/files#diff-882717ba3a21107bfb44676303dba09977ecbddf0b5894557c25c4facf820851R40